cg/store/status_collaboration_filters.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+from typing import List, Optional, Callable
+
+from sqlalchemy.orm import Query
+
+from cg.store.models import Collaboration
+
+
+def get_collaboration_by_internal_id(collaborations: Query, internal_id: str, **kwargs) -> Query:
```

Comment:
I think the naming we use for the filter functions is a little misleading. We are not returning the actual objects, but rather queries which filters the data by different properties. So maybe ```filter_collaboration_by_internal_id``` is a better name than ```get_collaboration_by_internal_id```. I think we should change this for all filter functions.

Reply:
Good idea, I'll change this one 👍

Reply:
I agree - I  will change in my ongoing customers PR